docroot/modules/humsci/hs_admin/hs_admin.module

```diff
@@ -0,0 +1,116 @@
+
+  if ($logged_in) {
+    $variables['#attached']['library'][] = 'hs_admin/hs_admin';
+  }
+}
+
+/**
+ * Implements hook_preprocess_html().
+ */
+function hs_admin_preprocess_html(&$variables) {
+  // Get currently active user and his roles.
+  $account = \Drupal::currentUser();
+  $roles = $account->getRoles();
+
+  foreach ($roles as $role) {
+    $variables['attributes']['class'][] = 'role-' . $role;
+  }
+}
+
+/**
+ * Implements hook_toolbar_alter().
+ */
+function hs_admin_toolbar_alter(&$items) {
+  $uid = \Drupal::currentUser()->id();
+  $user = User::load($uid);
+
+  if (!$user->hasPermission('show manage link') && $uid != 1) {
+    unset($items['admin_toolbar_tools']);
+    unset($items['administration']);
+  }
+}
+
+/**
+ * Implements hook_menu_local_tasks_alter().
+ */
+function hs_admin_menu_local_tasks_alter(&$data, $route_name, RefinableCacheableDependencyInterface &$cacheability) {
+  // Disable edit/create node tabs for publish and delete.
+  unset($data['tabs'][0]['entity.node.publish']);
+  unset($data['tabs'][0]['entity.node.delete_form']);
+}
+
+/**
+ * Implements hook_preprocess_details().
+ */
+function hs_admin_preprocess_details(&$variables) {
+  $expanded = ['menu settings'];
+  // Close all sidebar elements if not Menu Settings.
+  $title = $variables['title'] ?? '';
+  if (!is_array($title) && in_array(strtolower((string) $title), $expanded, TRUE)) {
+    $variables['attributes']['open'] = 'open';
+  }
+  else {
+    unset($variables['attributes']['open']);
+  }
+}
+
+/**
+ * Impelments hook_preprocess_form_element().
+ */
+function hs_admin_preprocess_form_element(&$variables) {
+  $name = $variables['name'] ?? '';
+  $type = $variables['element']['#type'] ?? '';
+  // Check that it's the published status field
+  // and check it for a label.
+  if ($type === 'item' && strtolower($name) === 'meta[published]' && isset($variables['children'])) {
+    $variables['children'] = Markup::create('Revisions');
+  }
+}
+
+/**
+ * Implements hook_form_BASE_FORM_ID_alter().
+ */
+function hs_admin_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Some tabs should always be first regardless of newly added tabs
+  // in node add/edit forms.
+  $form['menu']['#weight'] = -4;
+  $form['author']['#weight'] = -3;
+  $form['promote']['#weight'] = -2;
+  $form['options']['#weight'] = -2;
+  $form['path']['#weight'] = -1;
+  // Some tabs should always be last regardless of newly added tabs.
+  $form['url_redirects']['#weight'] = 96;
+  $form['rabbit_hole']['#weight'] = 97;
+  $form['xmlsitemap']['#weight'] = 98;
+  $form['meta']['#weight'] = 99;
+  $form['revision']['#weight'] = 100;
+  $form['revision_information']['#weight'] = 100;
+  // Weight is applied to some tabs after form build.
+  $form['#after_build'][] = 'hs_admin_form_after_build';
+}
+
+/**
+ * Implements hook_form_after_build().
+ */
+function hs_admin_form_after_build($form, FormStateInterface $form_state) {
```

Comment:
this is not a hook. it's a callback from above. adjust the comment and prepend the helper function with an underscore like: `function _hs_admin_node_form_after_build`

Reply:
Fixed.